File: app/src/main/java/com/haeti/ddolie/presentation/theme/Colors.kt
package com.haeti.ddolie.presentation.theme

import androidx.compose.runtime.staticCompositionLocalOf
import androidx.compose.ui.graphics.Color

val RedPrimary = Color(0xFFF44522)
val RedSecondary = Color(0x33FB4723)
val RedTertiary = Color(0x40FB4723)
val Green = Color(0xFF68EDCE)
val White = Color(0xFFFFFFFF)
val Black = Color(0xFF000000)

data class DdoLieColors(
	val redPrimary: Color = RedPrimary,
	val redSecondary: Color = RedSecondary,
	val redTertiary: Color = RedTertiary,
	val green: Color = Green,
	val white: Color = White,
	val black: Color = Black
)

val LocalDdoLieColors = staticCompositionLocalOf { DdoLieColors() }

File: watchos/DdoLie/Theme/Colors.swift
import SwiftUI

/// Mirrors wearos Colors.kt — ARGB hex values must stay identical.
/// Parity rules: .claude/rules/parity.md (checked by the parity-check skill).
enum DdoLieColors {
    static let redPrimary = Color(argb: 0xFFF44522)
    static let redSecondary = Color(argb: 0x33FB4723)
    static let redTertiary = Color(argb: 0x40FB4723)
    static let green = Color(argb: 0xFF68EDCE)
    static let white = Color(argb: 0xFFFFFFFF)
    static let black = Color(argb: 0xFF000000)
}

extension Color {
    /// ARGB literal in the same 0xAARRGGBB form used by Compose's Color(...)
    init(argb: UInt32) {
        self.init(
            .sRGB,
            red: Double((argb >> 16) & 0xFF) / 255.0,
            green: Double((argb >> 8) & 0xFF) / 255.0,
            blue: Double(argb & 0xFF) / 255.0,
            opacity: Double((argb >> 24) & 0xFF) / 255.0
        )
    }
}
